changelog/11942.bugfix.md

```diff
@@ -0,0 +1 @@
+Refactor form activation loop to fix double slot validation in a single request.
```

Comment:
I would like to rephrase this changelog to be more specific (some starter suggestion below), including a link to the Github ticket:
```suggestion
Fix calling the form validation action twice for the same user message triggering a form.
```